.github/workflows/rust.yml

```diff
@@ -290,6 +298,7 @@ jobs:
       # with a OS-dependent path.
       - name: Setup Rust toolchain
         run: |
+          rustup update stable
```

Comment:
Is the update here strictly necessary? Or should already be covered as part of the toolchain install below?

Reply:
i'll check it, maybe its not needed.